lib/packwerk/application_load_paths.rb

```diff
@@ -8,18 +8,25 @@ module ApplicationLoadPaths
     class << self
       extend T::Sig
 
-      sig { returns(T::Array[String]) }
-      def extract_relevant_paths
-        assert_application_booted
+      sig { params(root_path: String).returns(T::Array[String]) }
+      def extract_relevant_paths(root_path)
+        load_application(root_path) unless application_loaded?
         all_paths = extract_application_autoload_paths
         relevant_paths = filter_relevant_paths(all_paths)
         assert_load_paths_present(relevant_paths)
         relative_path_strings(relevant_paths)
       end
 
-      sig { void }
-      def assert_application_booted
-        raise "The application needs to be booted to extract load paths" unless defined?(::Rails)
+      sig { params(root_path: String).void }
+      def load_application(root_path)
+        require File.expand_path(File.join(root_path, "config/environment"))
```

Comment:
@wildmaples Now that ApplicationLoadPath loads the Rails app, does `bin/packwerk` still need the following line?
https://github.com/Shopify/packwerk/blob/373993a437aeca2223186617a95f09f2e57cee29/lib/packwerk/generators/templates/packwerk#L18

Reply:
Good idea - I've just checked against one of our applications and it can be removed from the `bin/packwerk` file 👍  I'll update it in the template now

Reply:
This has the nice effect of meaning `bin/packwerk check` no longer loads Rails